runtime/lua/vim/lsp/handlers.lua

```diff
@@ -409,7 +409,7 @@ for k, fn in pairs(M) do
     })
 
     if err then
-      error(tostring(err))
+      return err_message(tostring(err))
```

Comment:
Is it confusing to return early like this when err_message has no return value?

Reply:
it triggers an error if it doesn't stop there. Returning a nil is ok IMO ?